### PR TITLE
fix: Pin dbt-core to v1.8.7

### PR DESCRIPTION
**Pull request by k-rheinheimer**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -39,7 +39,6 @@ endif
 .PHONY: gen-single-site
 gen-single-site:
 	cloudquery migrate transformations/$(transformation_dir)/tests/postgres.yml
-	dbt seed --target dev-pg --profiles-dir transformations/$(transformation_dir)/tests --project-dir transformations/$(transformation_dir)
```
Comment (erezrokah) [inline Makefile:42]: I think for the cost policies we need the seed, but whatever works

**Review by erezrokah — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (k-rheinheimer) [reply Makefile:42]: the cost policies tests worked without it but i don't think that fixed what I was trying to do anyway - pinning the version works for the test steps but for some reason when running through the makefile command it's still running the non-working version. any idea on how to make the makefile either respect the requirements.txt or add specific version use to the makefile?

Comment (erezrokah) [reply Makefile:42]: I think you need to replace https://github.com/cloudquery/policies/blob/1bf85662c30f85cc5d2365a3906c1c96f5e97223/.github/workflows/transformations_docs_deploy.yml#L47 with pip install

**Review by erezrokah — APPROVED** [review]

**Comment by erezrokah** [comment]
Looks like the bucket for the S3 tests was created on our `cq-playground` account so it got deleted. I re-created it on the `cq-integration-tests` account and gave the necessary permissions for the GitHub Action to access it.

**Comment by erezrokah** [comment]
OK I'm going to force merge this as it fixes everything expect Athena. We want to unblock https://github.com/cloudquery/policies/pull/1090 and we can fix Athena later

**Pull request merged**